### PR TITLE
1696 upgrade react native etc

### PR DESCRIPTION
**Pull request by edmofro**

### CODE REVIEW

File: packages/meditrak-app/app/authentication/actions.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const login = (emailAddress, password) => async (
   const loginCredentials = {
     emailAddress,
     password,
-    deviceName: DeviceInfo.getDeviceName(),
+    deviceName: await DeviceInfo.getDeviceName(),
```
Comment (chris-bes) [inline packages/meditrak-app/app/authentication/actions.js:52]: I guess this is why it comes back as unknown for you before?

**Review by chris-bes — COMMENTED** [review]
This looks fine to me but to be honest I'm not very confident since it has a lot of stuff that I'm still very green :(. Wouldn't mind if this gets another pair of eyes on it...

File: packages/meditrak-app/app/authentication/actions.js (hunk 1 above)
Comment (edmofro) [reply packages/meditrak-app/app/authentication/actions.js:52]: No the `await` was in place when I reported that. Still not sure, I think it's just the Genymotion device I have set up doesn't give a name.

**Comment by edmofro** [comment]
> This looks fine to me but to be honest I'm not very confident since it has a lot of stuff that I'm still very green :(. Wouldn't mind if this gets another pair of eyes on it...

I think that's fine tbh @chris-bes - anyone at BES will be great to this stuff, so your approval is enough for me given how busy the rest of the team is! :-)

**Review by chris-bes — APPROVED** [review]
Did the last smoke test here (https://github.com/beyondessential/tupaia-backlog/issues/1696#issuecomment-742857175). Approve!

**Pull request merged**